### DISCUSSION TITLE
The new achievements

### DISCUSSION THREAD

**Discussion by temirovazat** [community/community#18225] [Other Feature Feedback, Questions, & Ideas]
the new achievements look very cool and creative, how do you like the design and what do you think about the new achievements. Write in the answers

**Comment by zynowhere** [comment]
[https://github.com/Schweinepriester/github-profile-achievements](url)

Unfortunately, Arctic Code Vault Contributor and Mars 2020 Contributor can not be obtained, I wonder if there will be a 2022 or later version

**Comment by mribeirodantas** [comment]
I think it's a nice feature. I wonder if it'll turn into something like Stack Exchange where users can have hundreds of [sometimes even repeated] badges or will be limited to only a few badges. Besides, I think it would be more interesting to have badges highlighting specific events than a number of everyday actions.

Getting a badge for having an answer accepted for a question that has remained without an answer for over a year would mean a lot! Getting a badge for having X pull requests merged seems... meh.
